slick-testkit/src/codegen/scala/slick/test/codegen/GenerateRoundtripSources.scala

```diff
@@ -65,6 +65,16 @@ class Tables(val profile: JdbcProfile){
   }
   val SingleNonOptionColumn = TableQuery[SingleNonOptionColumn]
 
+  /** Tests table name cyclic dependency */
+  case class ColumnRow(id: Long , name: String  )
+  class Column(tag: Tag) extends profile.api.Table[ColumnRow](tag, "column") {
```

Comment:
What is the `Column` for here? The creation of table `Table` in `h2.sql` alone would not suffice?